supabase/functions/_backend/private/plugins/stats.ts

```diff
@@ -212,7 +204,7 @@ async function post(c: Context, body: AppStats) {
       }, 200)
     }
     rows.push(stat)
-    await Promise.all([sendDevice(c, device).then(() => sendStats(c, rows))])
+    await Promise.all([sendDevice(c, device), sendStats(c, rows)])
```

Comment:
i think the then was for a reason, but i couldn't find why anymore, so we can try

Reply:
maybe it was for the stats it was spamming otherwise